qemu/tests/win_msft_sign_check.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    sigverif test:
+    1) Boot guest with related virtio devices
+    2) Run driver signature check command in guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+
+    driver_name = params["driver_name"]
+    session = utils_test.qemu.windrv_check_running_verifier(session, vm,
+                                                            test, driver_name)
+
+    # check if Windows VirtIO driver is msft digital signed.
+    device_name = params["device_name"]
+    chk_cmd = params["vio_driver_chk_cmd"] % device_name[0:30]
```

Comment:
@lijinlijin Would you introduce why `device_name[0:30]` instead of `device_name`?

Reply:
Because driverquery returned DeviceName is not full, it's cut off and can only display device_name[0:30]